docs/guides/cli.md

```diff
@@ -1 +1,174 @@
-# Coming Soon
+
+# GuideLLM CLI User Guide 
+
+For more details on setup and installation, see the Setup and [Installation](https://github.com/neuralmagic/guidellm?tab=readme-ov-file#installation) sections. 
+
+## About GuideLLM
+
+The GuideLLM CLI is a performance benchmarking tool to enable you to evaluate and visualize your LLM inference serving performance before your deploy to production. GuideLLM runs via the CLI and takes in a wide array of input arguments to enable you to configure your workload to run benchmarks to the specifications that you desire. This ultimately provides the ability to understand bottlenecks in your inference serving pipeline and make changes before your users are effected by your LLM application. 
+
+## GuideLLM CLI Quickstart 
+
+#### 1. Start an OpenAI Compatible Server (vLLM)
+
+GuideLLM requires an OpenAI-compatible server to run evaluations. It's recommended that [vLLM](https://github.com/vllm-project/vllm) be used for this purpose. To start a vLLM server with a Llama 3.1 8B quantized model, run the following command:
+
+```bash
+vllm serve "neuralmagic/Meta-Llama-3.1-8B-Instruct-quantized.w4a16"
+```
+
+For more information on starting a vLLM server, see the [vLLM Documentation](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html).
+
+#### 2. Run a GuideLLM Evaluation
+
+To run a GuideLLM evaluation, use the `guidellm` command with the appropriate model name and options on the server hosting the model or one with network access to the deployment server. For example, to evaluate the full performance range of the previously deployed Llama 3.1 8B model, run the following command:
+
+```bash
+guidellm \
+  --target "http://localhost:8000/v1" \
+  --model "neuralmagic/Meta-Llama-3.1-8B-Instruct-quantized.w4a16"
```

Comment:
instruction tuned models have a different behavior than non-instruction tuned models. They require specific tags in messages. This is described in the [Llama 3.1 Prompt Guide](https://llama.meta.com/docs/model-cards-and-prompt-formats/llama3_1). It isn't clear if `--target` is the base url, or the completions API for foundation models, or the base url for chat appllications.

For instruction tuned models, it is ideal to use a chat completion api. The model will still work correctly without, but is less optimal, and chat turns cannot be encoded in the message. If guidellm only has one turn, it may not matter.